CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Fixed issue in `CMakePresets.json` where Ninja presets were broken
+- Removed cycle in MAPL_Generic.F90
```

Comment:
This is not a very useful note for the ChangeLog.   Users can look at "diff" to see the deleted line.   Here we should explain the why?  What was breaking what were the symptoms or consequences.

Reply:
That line was added by me. After thinking twice, I believe that "I" and "J" are from different sources and should not be cycled when they are the same.

Reply:
@weiyuan-jiang You can probably remove this line and then edit line 19:
```
- A small performance improvement. cycle => exit in MAPL_Generic.F90
```
to be whatever you think this PR plus the first on should reflect.

Reply:
Yeah. I'll just remove this and add "skip changelog".